server/src/instant/util/string.clj

```diff
@@ -24,6 +24,10 @@
     (.update ^CRC32 crc bytes)
     (.getValue ^CRC32 crc)))
 
+(defn safe-trim [s]
+  (and (string? s)
```

Comment:
Maybe this should be `when` so that you return null instead of `false` from safe-trim?

```suggestion
  (when (string? s)
```